Menu/MenuBuilder.php

```diff
@@ -52,16 +47,15 @@ class MenuBuilder
     private $eventDispatcher;
 
     /**
-     * Constructor.
-     *
      * @param Pool                     $pool
      * @param FactoryInterface         $factory
      * @param MenuProviderInterface    $provider
      * @param EventDispatcherInterface $eventDispatcher
+     *
+     * @deprecated first argument is deprecated since 2.4. To be removed in 3.0
```

Comment:
can you explain WHAT is deprecated?

if i would read this in a logfile i wouldn't know what to do